azure-pipelines.yml

```diff
@@ -0,0 +1,209 @@
+trigger:
+  branches:
+    include:
+    - master
```

Comment:
That might depend on just how different your pipelines would be between the two branches. But maintaining multiple pipelines tends to be more work, and usually they have quite a bit in common. So I would recommend we start with one pipeline that handles both master and dev, and we add conditions or variables as necessary to get whatever distinct behavior you want between the two branches.

Reply:
I just took a look at your `dotnet-apiport-master-Release` pipeline. It seems to me that it's distinct from the dev-CI pipeline in that it's meant for *releases* rather than a per-branch difference. 
So building master with this new YML file should be fine. But as a follow-up to this PR, we can take another look at the master-Release pipeline and integrate its unique abilities into this YAML so that we can build releases as well as handle PR/CI builds and eventually retire the master-Release pipeline too.

Reply:
Sounds good plan